receiver/hostmetricsreceiver/README.md

```diff
@@ -77,10 +86,9 @@ network:
 
 ```yaml
 process:
-  disk:
```

Comment:
Is this change on purpose?

Reply:
I thought it was an error but I can cancel this change because it is out of the scope of this PR

Reply:
Better for a separate PR.

Reply:
ok I dropped this change ;)